tensorflow_addons/layers/normalizations.py

```diff
@@ -66,15 +70,12 @@ class GroupNormalization(tf.keras.layers.Layer):
 
     Output shape:
         Same shape as input.
-
```

Comment:
I removed this as this way of referencing did not seem to be consistent with citations in other functions. It also makes it difficult to find the reference.